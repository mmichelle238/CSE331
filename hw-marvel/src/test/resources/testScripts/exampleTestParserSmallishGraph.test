# Test the parser on a graph that contains several nodes
# with two edges each.

LoadGraph heroes staffSuperheroes.tsv
ListChildren heroes Ernst-the-Bicycling-Wizard