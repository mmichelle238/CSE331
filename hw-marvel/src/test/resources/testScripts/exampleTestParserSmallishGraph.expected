# Test the parser on a graph that contains several nodes
# with two edges each.

loaded graph heroes
the children of Ernst-the-Bicycling-Wizard in heroes are: Grossman-the-Youngest-of-them-all(CSE331) Notkin-of-the-Superhuman-Beard(CSE331) Notkin-of-the-Superhuman-Beard(CSE403) Perkins-the-Magical-Singing-Instructor(CSE331)